### ISSUE TITLE
Emulator crashes when V or U are out of range

### ISSUE THREAD

**Issue by tcstewar** [nengo/nengo-loihi#88]
`V` and `U` are both bounded in Loihi, but right now if you go outside of these bounds in the emulator, then an exception is raised. 

For example:
```
model = nengo.Network(seed=3)
with model:
    stim = nengo.Node(1)
    a = nengo.Ensemble(100, 1)
    nengo.Connection(stim, a)
    b = nengo.Ensemble(100, 1)
    nengo.Connection(a, b, transform=10, synapse=0.1)
```

When run, this raises this exception:
```
Traceback (most recent call last):
  File "c:\users\terry\documents\github\nengo-loihi\nengo_loihi\simulator.py", line 409, in run_steps
    self.simulator.run_steps(steps)
  File "c:\users\terry\documents\github\nengo-loihi\nengo_loihi\loihi_cx.py", line 740, in run_steps
    self.step()
  File "c:\users\terry\documents\github\nengo-loihi\nengo_loihi\loihi_cx.py", line 693, in step
    raise SimulationError("Overflow in V (max was %d)" % self.v.max())
nengo.exceptions.SimulationError: Overflow in V (max was 8459846)
```

I believe the solution should be to clip V and U, rather than raising exceptions in `loihi_cx.py`:
```python
        if np.any(u2 > U_MAX):
            raise SimulationError("Overflow in U (max was %d)" % u2.max())
        if np.any(u2 < U_MIN):
            raise SimulationError("Underflow in U (min was %d)" % u2.min())

        # self.V[:] = self.decayV_fn(v, self.decayV, a=12) + u2
        self.v[:] = self.decayV_fn(self.v, u2)
        if np.any(self.v > V_MAX):
            raise SimulationError("Overflow in V (max was %d)" % self.v.max())
        if np.any(self.v < V_MIN):
            raise SimulationError("Underflow in V (min was %d)" % self.v.min())
```

**Comment by tbekolay** [comment]
We added this in 54954fb035c57ecaf48603cbb58738099a0b273b because we were getting very strange behavior on some models on hardware, which turned out to be because some weights were very large and was causing overflows. The explicit exceptions allow us to notice this for debugging early on rather than seeing weird, difficult to debug behavior on hardware.

**Comment by tbekolay** [comment]
I mean, any model that raises this exception is unlikely to run correctly on hardware so it depends on whether that matters for these particular examples... @hunse added the check and exception, any thoughts about other ways we can avoid overflows?

**Comment by tbekolay** [comment]
I sent you a slack message with a plot for what I have seen happen for voltage overflows. I'm fine with changing to a warning if @hunse is :+1:

**Comment by tbekolay** [comment]
Actually, what might be nicer than just a warning would be for the emulator to raise a warning by default but have a flag so that these warnings can be raised as errors for testing and whatnot. I'll make a PR for that tomorrow.

**Comment by tbekolay** [comment]
Fixed in #91.
